apps/website/src/examples/InputGroup.radioGroup.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { InputGroup, Radio } from '@itwin/itwinui-react';
+import { SvgSmileyHappy } from '@itwin/itwinui-react/cjs/core/utils';
+import { SvgSmileySad } from '@itwin/itwinui-icons-react';
+import * as React from 'react';
+
+export default (args) => {
+  const option1Label = <SvgSmileyHappy />;
+  const option2Label = <SvgSmileySad />;
+  function action(arg0: string): React.ChangeEventHandler<HTMLInputElement> {
+    throw new Error('Function not implemented.');
+  }
+
+  return (
+    <>
+      <InputGroup label='Radio group' message='Tell me how happy you are' {...args}>
+        <Radio
+          name='choice'
+          value='option1'
+          onChange={action('Clicked option 1!')}
+          label={option1Label}
+          disabled={args.disabled}
+          required={args.required}
+        />
+        <Radio
+          name='choice'
+          value='option2'
+          onChange={action('Clicked option 2!')}
+          label={option2Label}
+          disabled={args.disabled}
+          required={args.required}
+        />
+      </InputGroup>
+    </>
+  );
+};
```

Comment:
So the website is broken when I try to view this. The issue is with the import for `SvgSmileyHappy`. It also needs to be imported the same way that the sad face is. There's also some extra things from storybook that aren't needed in this example.

I think you can just change this example to this:
```suggestion
/*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
import { InputGroup, Radio } from '@itwin/itwinui-react';
import { SvgSmileyHappy, SvgSmileySad } from '@itwin/itwinui-icons-react';

export default () => {
  return (
    <>
      <InputGroup label='Radio group' message='Tell me how happy you are'>
        <Radio name='choice' value='option1' label={<SvgSmileyHappy />} />
        <Radio name='choice' value='option2' label={<SvgSmileySad />} />
      </InputGroup>
    </>
  );
};
```